docs/components.md

```diff
@@ -54,7 +54,6 @@ The distribution offers support for the following components.
 | [otlp](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)                                                                  | [stable]         |
 | [postgresql](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/postgresqlreceiver)                                              | [beta]           |
 | [prometheus](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver)                                              | [beta]           |
-| [prometheus_simple](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/simpleprometheusreceiver)                                 | [beta]           |
```

Comment:
If anyone has strong preferences the other way I'm happy to remove the `simpleprometheus` entry instead.

Reply:
We are using the component names as they are applied in the config for others. So I think we should remove `simpleprometheus` instead

Reply:
Makes sense, I've swapped.